### ISSUE TITLE
Delete a tokenId mapped index after popping it from tokenIds array

### ISSUE THREAD

**Issue by siriusbeans** [Kromatika-Finance/limit-trade#17]
Within _stopMonitor(), after swapping tokenId array positions in removeElementFromArray(), a new array index is assigned to the shifted tokenId.  The tokenId that was popped from the array still has an index mapped within the tokenIndexPerTokenId mapping. Should this index be deleted from the mapping **after** this line:

https://github.com/Kromatika-Finance/limit-trade/blob/3113669772b30f00b887e8429ed313724885c65f/contracts/LimitOrderMonitor.sol#L240

Example:
`delete tokenIndexPerTokenId[_tokenId];`

**Comment by Aristokrates** [comment]
Yes, you are right, the tokenId has been deleted from the array, but not from the mapping. 
And it should be deleted from the mapping as well.
